configs/myconfigs/sample.py

```diff
@@ -289,6 +302,7 @@ def handler(signum, frame):
 
         if exit_event.getCause() == "work started count reach":
             simulateROI(system, roiInstructions, samples)
+            break       #  exit after ROI is complete
```

Comment:
My original idea here is that you would always run to the end of the application, instead of breaking after the ROI. Do you see a need to change this?